qcodes/utils/installation_info.py

```diff
@@ -47,7 +46,22 @@ def get_qcodes_version() -> str:
     """
     Get the version of the currently installed QCoDeS
     """
-    return qcodes.__version__
+    import qcodes
+    package_name = "qcodes"
+
+    qcodes_path = Path(qcodes.__file__).parent
+    if _has_pyproject_toml_and_is_git_repo(qcodes_path.parent):
```

Comment:
ok, so this becomes the only project-specific piece and the rest will be the same when we want to expand this to other packages, right? that's good, i think :)

Reply:
Yes basically if you should check .parent or .parent.parent